lib/mutgen.c

```diff
@@ -292,6 +294,101 @@ mutation_matrix_free(mutation_model_t *self)
     return 0;
 }
 
+/***********************
+ * SLiM mutation model */
+
+/* from slim_globals.h */
+typedef int64_t slim_mutationid_t;		// identifiers for mutations, which require 64 bits since there can be so many
+
+/* Copy of MutationMetadataRec in slim_sim.h: */
+typedef struct __attribute__((__packed__)) {
+    int32_t mutation_type_id_; // 4 bytes (int32_t): the id of the mutation type the mutation belongs to
```

Comment:
We can get rid of the trailing underscores here I'd say, might as well use the local coding conventions. A bit more details on the provenance would be help here I think - what date/what git hash were these structs copied at?

Reply:
I'll put in more details to the reference. But, I guess I vote to keep them exactly matching, as a reminder that we're copying this from elsewhere (so it's not to be changed).

Reply:
OK, that sounds good.